src/components/EntryForm.js

```diff
@@ -208,13 +203,13 @@ const EntryForm = ({
               ref={register}
               name="isFree"
               type="checkbox"
-              checked={!isPaymentRequired}
+              checked={!noPayment}
               className={styles.feeToggle}
-              onChange={(event) => setIsPaymentRequired(!event.target.checked)}
+              onChange={(event) => setNoPayment(!event.target.checked)}
             />
           </label>
 
-          {isPaymentRequired && (
+          {noPayment && (
```

Comment:
I think 206, 208 and 212 should be inverted again

Reply:
Yes. Done